integration_test/third_party_apps_data/agent/ops-agent/linux/enable_postgresql

```diff
@@ -0,0 +1,45 @@
+# Configures Ops Agent to collect telemetry from the app and restart Ops Agent.
+
+set -e
+
+sudo tee /etc/google-cloud-ops-agent/config.yaml > /dev/null << EOF
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+metrics:
+  receivers:
+    postgresql_metrics:
+      type: postgresql
+      collection_interval: 30s
+      username: postgres
```

Comment:
Does Postgres require username and password out of the box?

Reply:
Out of the box behavior differs for different distributions. Debian/CentOS is Ident based (logged in user) auth for unix socket, password auth via local TCP, password via external TCP. SLES is Ident based for unix of local TCP, no auth via external TCP.

Reply:
😵‍💫